_includes/team-signup.html

```diff
@@ -7,31 +9,72 @@
       novalidate>
     <div id="mc_embed_signup_scroll">
       <div class="mc-field-group">
-        <label for="mce-EMAIL">Email Address</label>
-        <input type="email" value="" name="EMAIL" class="required email full-width" id="mce-EMAIL">
+        <label for="mce-EMAIL">Email Address </label>
+        <input type="email" value="" name="EMAIL" class="required email" id="mce-EMAIL">
       </div>
       <div class="mc-field-group">
-        <label for="mce-FNAME">Your First Name</label>
-        <input type="text" value="" name="FNAME" class="" id="mce-FNAME" size="24">
+     <label for="mce-FNAME">First Name </label>
+     <input type="text" value="" name="FNAME" class="" id="mce-FNAME">
       </div>
       <div class="mc-field-group">
-        <label for="mce-LNAME">Your Last Name</label>
-        <input type="text" value="" name="LNAME" class="" id="mce-LNAME" size="24">
-      </div>
-      <div class="mc-field-group">
-        <label for="mce-MMERGE4">Organisation</label>
-        <p class="explanation">If your team represents a school, or another organisation, enter its name here.</p>
-        <input type="text" value="" name="MMERGE4" class="full-width" id="mce-MMERGE4">
-      </div>
-      <div id="mce-responses" class="clear">
-        <div class="response" id="mce-error-response" style="display:none"></div>
-        <div class="response" id="mce-success-response" style="display:none"></div>
-      </div>
-      <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_1af17c2a3426de258620aae46_dd7d04139d" tabindex="-1" value=""></div>
-      <div class="clear">
-        <input class="button button-primary" type="submit" value="Register your interest" name="subscribe" id="mc-embedded-subscribe">
-        <span>or</span>
-        <a class="button" href="mailto:{{ site.emails.teams }}" target="_blank">Speak to someone</a>
+     <label for="mce-LNAME">Last Name </label>
+     <input type="text" value="" name="LNAME" class="" id="mce-LNAME">
       </div>
+<div class="mc-field-group size1of2">
+        <label for="mce-PHONE">Phone Number </label>
```

Comment:
Do we really need to ask for phone number and postal address here?

As I understand it, this form is for people who are vaguely interested and would like to know more, rather than a full commitment to leading a team soon.

Given that, this level of detail feels like to put people off.

Reply:
Currently all of these fields are set as 'required' in mailchimp, so I can't change this here.

@jimmythompson is there any reason why the phone number and address aren't optional?

Reply:
Oops just looked into this, the phone number is actually optional, the point still stands for address however